drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -860,9 +860,9 @@ func (k *k8sSchedOps) GetRemotePXNodes(destKubeConfig string) ([]node.Node, erro
 	return remoteNodeList, nil
 }
 
-// getContainerPVCMountMap is a helper routine to return map of containers in the pod that
+// GetContainerPVCMountMap is a helper routine to return map of containers in the pod that
```

Comment:
Why is this needed?

Reply:
This function was private, I had to make it public to use it in our helper functions.

Reply:
Got it but one question here is do we need to submit the same fix in schedops as well? if not let's say I vendor in the latest changes in schedops will it revert this fix?

Reply:
This is not vendored in code. This is Torpedo code itself.

Reply:
Okay sounds good.